src/Centreon/Domain/Repository/NagiosServerRepository.php

```diff
@@ -51,7 +53,82 @@ class NagiosServerRepository extends ServiceEntityRepository
      */
     public function checkListOfIds(array $ids): bool
     {
-        return $this->checkListOfIdsTrait($ids, NagiosServer::TABLE, NagiosServer::ENTITY_IDENTIFICATOR_COLUMN);
+        return $this->checkListOfIdsTrait($ids);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getPaginationList($filters = null, int $limit = null, int $offset = null, $ordering = []): array
+    {
+        $collector = new StatementCollector;
+
+        $sql = 'SELECT SQL_CALC_FOUND_ROWS * '
+            . 'FROM `' . $this->getClassMetadata()->getTableName() . '`';
+
+        if ($filters !== null) {
```

Comment:
remove this condition

Reply:
sometimes we need to get the list without any filters to be applied